app/connectors/CalculationDetailsConnector.scala

```diff
@@ -19,12 +19,13 @@ package connectors
 import config.AppConfig
 import connectors.httpParsers.CalculationDetailsHttpParser.{CalculationDetailResponse, CalculationDetailsHttpReads}
 import play.api.Logging
-import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpReads, HttpResponse}
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.http.{HeaderCarrier, StringContextOps}
 
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
-class CalculationDetailsConnector @Inject()(httpClient: HttpClient,
+class CalculationDetailsConnector @Inject()(httpClient: HttpClientV2,
```

Comment:
Hi, I noticed there are no unit tests for this change, is it okay not have unit tests for this type of change? I'm asking because I making the same change for OptIn code, thanks